cybersixgill_actionable_alerts/manifest.json

```diff
@@ -0,0 +1,72 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b27feb80-b06f-4200-981a-e91a031d62e6",
+  "app_id": "cybersixgill-actionable-alerts",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor activity in all of our sources regarding key company assets and provide real-time alerts on incoming threats, to create dedicated playbooks",
```

Comment:
Can you specify what "our sources" refers to?

Reply:
Here Sources refer to Dark Web